src/sort.c

```diff
@@ -425,11 +425,11 @@ void sortCommandGeneric(client *c, int readonly) {
 
             ln = zsl->tail;
             if (start > 0)
-                ln = zslGetElementByRank(zsl,zsetlen-start);
+                ln = zslGetElementByRank(zsl->header,zsl->level-1,zsetlen-start);
```

Comment:
let's make a one line wrapper function that takes a `zskiplist*` so that we don't need to make this change, only advanced users need to worry about the level argument.